e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -446,6 +446,19 @@ def _get_climo_dataset(self, season: str) -> xr.Dataset:
         if "slat" in ds.dims:
             ds = ds.drop_dims(["slat", "slon"])
 
+        all_vars = list(ds.data_vars.keys())
+        keep_bnds = [var for var in all_vars if "bnd" in var]
```

Comment:
```suggestion
        keep_bnds = [var for var in all_vars if "bnd" or "bound" in var]
```

Should we also consider `"bound"` here?

Reply:
Addressed in [`1bf91a6` (#798)](https://github.com/E3SM-Project/e3sm_diags/pull/798/commits/1bf91a6ca085255acc13f8affee88d369703d01a)